pkg/proxy/iptables/proxier.go

```diff
@@ -932,7 +1074,8 @@ func (proxier *Proxier) syncProxyRules() {
 			for _, endpointChain := range endpointChains {
 				writeLine(natRules,
 					"-A", string(svcChain),
-					"-m", "comment", "--comment", svcName.String(),
+					"-m", "comment", "--comment",
+					fmt.Sprintf(`"Session affinity rules for %s (svc-guid: %s)"`, svcName.String(), svcInfo.guid),
```

Comment:
I don't think the expanded comment is needed, or at least could come in a different PR.  Strip everything that isn't needed to get this PR through